pkg/apis/nfd/v1alpha1/expression.go

```diff
@@ -24,6 +24,7 @@ import (
 	"strconv"
 	"strings"
 
+	"golang.org/x/exp/maps"
```

Comment:
I am confused, didn't we just got rid of this pkg? 
https://github.com/kubernetes-sigs/node-feature-discovery/pull/1502

Reply:
Indeed, we did 😇 But the non-exp `maps` package doesn't provide the `Keys()` and `Values()` funcs which we use here (and elsewhere in this PR